samples: video: capture: Fix floating point logging issue

ngphibang opened the pull request:
The default frame rate is printed as a floating point number. Without enabling CONFIG_REQUIRES_FLOAT_PRINTF, it is not printed at all. Fix it.

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/80653

kartben reviewed (changes requested):
I believe the right fix is simply to set `CONFIG_REQUIRES_FLOAT_PRINTF=y` in the prj.conf

ngphibang committed:
samples: video: capture: Fix floating point logging issue

The default frame rate is printed as a floating point number. Without
enabling CONFIG_REQUIRES_FLOAT_PRINTF, it is not printed at all. Fix it.

Signed-off-by: Phi Bang Nguyen <phibang.nguyen@nxp.com>

ngphibang commented:
@kartben : That's right. Thank you for showing me this config that I was not aware of.

kartben reviewed (approved):
Thanks!
